x-pack/plugin/watcher/src/main/java/org/elasticsearch/xpack/watcher/rest/action/RestWatchServiceAction.java

```diff
@@ -30,12 +30,17 @@ public RestWatchServiceAction(Settings settings, RestController controller) {
 
         // @deprecated Remove in 6.0
         // NOTE: we switched from PUT in 2.x to POST in 5.x
+        // NOTE: We added back the old URL with the new VERB (POST) since we are deprecating _xpack/* URIs in 7.0
```

Comment:
im not 100% sure i did this right. Should I instead put both the PUT and POST back to non deprecated? In 7, only `POST /_watcher` and `POST /_xpack/watcher` exist, so this is why I did not undeprecate the `PUT /_watcher`